package.yaml

```diff
@@ -20,7 +20,7 @@ default-extensions:
 
 dependencies:
   - base >= 4.9 && < 5
-  - protolude >= 0.2
+  - protolude >= 0.3
```

Comment:
Would it be good to pin this to 0.3.x in case there's other breaking changes in a future 0.N, N>3?